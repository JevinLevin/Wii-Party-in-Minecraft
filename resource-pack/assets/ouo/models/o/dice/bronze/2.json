{
	"credit": "OuO Studio Creation",
	"parent": "ouo:o/dice/parent/2",
	"textures": {
		"1": "ouo:o/dice/bronze",
		"particle": "ouo:o/dice/bronze"
	}
}