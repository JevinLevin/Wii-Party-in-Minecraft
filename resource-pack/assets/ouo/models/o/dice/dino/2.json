{
	"credit": "OuO Studio Creation",
	"parent": "ouo:o/dice/parent/2",
	"textures": {
		"1": "ouo:o/dice/dino",
		"particle": "ouo:o/dice/dino"
	}
}