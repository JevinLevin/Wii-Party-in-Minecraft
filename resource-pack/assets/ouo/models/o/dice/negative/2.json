{
	"credit": "OuO Studio Creation",
	"parent": "ouo:o/dice/parent/2",
	"textures": {
		"1": "ouo:o/dice/negative",
		"particle": "ouo:o/dice/negative"
	}
}