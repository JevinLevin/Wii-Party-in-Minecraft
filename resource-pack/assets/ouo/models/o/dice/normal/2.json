{
	"credit": "OuO Studio Creation",
	"parent": "ouo:o/dice/parent/2",
	"textures": {
		"1": "ouo:o/dice/white",
		"particle": "ouo:o/dice/white"
	}
}