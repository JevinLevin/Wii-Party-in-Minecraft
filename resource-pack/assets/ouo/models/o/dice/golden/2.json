{
	"credit": "OuO Studio Creation",
	"parent": "ouo:o/dice/parent/2",
	"textures": {
		"1": "ouo:o/dice/golden",
		"particle": "ouo:o/dice/golden"
	}
}